### DISCUSSION TITLE
Package to just write a C++ program

### DISCUSSION THREAD

**Discussion by fl0wm0ti0n** [nanovms/ops#1334] [Q&A]
Hello, im curious if there is a package wich is made for c++ developing.

i just found the mongodb package which is named with "C++" runtime and language.

any advise? 

As i understand i need first a fitting package that i can write my app?:
_"Packages are officially supported images with pre-compiled languages (ie nodejs and php) or applications (ie Redis and Nginx). This makes it easier and more convenient to execute code without needing to compile your own interpreter(s)."_

**Comment by eyberg** [comment]
not generally no, if you have a c++ program I'd just use 'ops run myprogram' and it'll probably just work - if you need extra configuration you can put that in your config.json file - check out these examples: https://github.com/nanovms/ops-examples/tree/master/cpp
